hazelcast-hibernate/hazelcast-hibernate3/src/test/java/com/hazelcast/hibernate/RegionFactoryDefaultSlowTest.java

```diff
@@ -61,4 +65,19 @@ public void testQueryCacheCleanup() {
         assertEquals(numberOfEntities - evictedItemCount, queryRegion.getCache().size());
     }
 
+    @Test
+    public void testUpdate() {
+        insertDummyEntities(1);
+        Session session = sf.openSession();
+        Transaction tx = session.beginTransaction();
+        DummyEntity ent = (DummyEntity) session.get(DummyEntity.class, 0L);
```

Comment:
I would introduce a consistant for it. e.g start id. 

DummyEntity ent = (DummyEntity) session.get(DummyEntity.class, START_ID);